src/components/DpCard/DpCard.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta } from '@storybook/addon-docs'
+import DpCard from './DpCard'
+
+<Meta
+    title="Components/Card"
+    component={DpCard}
+/>
+
+# Card
+
+The DpCard component used to add Card section with `heading`, `heading-tooltip` props, and include some text or component
+like `dp-button` or `dp-loading`.
```

Comment:
Let's focus on the use case instead of the implementation here:

```suggestion
The Card component is used to present concise and structured information in a visually appealing and organized manner. It can be used to display widgets in a dashboard, or a collection of distinct items.

The heading and tooltip of a card is set via props, while the content is passed via its default slot.
```